lib/python/qmk/cli/list/keymaps.py

```diff
@@ -0,0 +1,84 @@
+"""List the keymaps for a specific keyboard
+"""
+import os
+import re
+import glob
+from bs4 import UnicodeDammit
```

Comment:
Right, I got that code from the qmk-compiler and forgot to add bs4 as a dependency in setup.py 
Ideally, these stuff should be moved to a helper library, so all Python code can access it. 
For the time being, install the module with pip for testing.

Reply:
Solution for any other prospective testers (or me because there's the chance I forget):

```diff
diff --git a/requirements.txt b/requirements.txt
index f6257e399..db936c33f 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -2,4 +2,5 @@
 # milc FIXME(skullydazed): Included in the repo for now.
 appdirs
 argcomplete
+bs4
 colorama
```

then

```
pip3 install -r requirements.txt
```

Anyway, this seems to work fully after making the above changes.

Reply:
Fixed it.

Reply:
We need to remove this dependency. The reason it's there for the API does not apply here, it would be better if someone fixed their inconsistent encoding when it's encountered. In general we're trying to keep the number of 3rd part modules low.

Reply:
I agree, although my intention was to use the pre-existing python code. In a following issue/PR I'd like to check and suggest a way of moving most of (all) common code and "business-logic" to the QMK python module so they can be shared between subprojects. Imho, the ideal endgame would be for the the CLI to be just a wrapper around the QMK library.